CONTRIBUTING.adoc

```diff
@@ -11,7 +12,31 @@ submit your pull request.
 If you have any questions about a possible submission, feel free to open
 an issue too.
 
-== Contributing to the terraform-oci-oke repository
+== Coding conventions
+
+Be sure to check {uri-coding-conventions}[the coding conventions] of this project before starting to develop.
+
+== Documentation you need to update
+
+A code contribution is as valuable as the related documentation that comes with it.
+
+Please update the documentation for the code you propose. The maintainers will love you, and your contribution will probably be accepted faster than if there is no or missing documentation.
+
+This is the actual "touch points" you need to pay attention:
+
+- Update CHANGELOG under the 'unreleased' section. Maintainer will reaffect your entry to another section in the release process
+- If your contribution changes fundamental aspects of the module, update the README introduction section
+- Add documentation for new variable in docs/terraformoptions.adoc
+- Add your GitHub handler to CONTRIBUTORS under the CONTRIBUTORS section
+- examples folder! Try to keep it simple. Update both main.tf and variables.tf
+- Do not forget to also update the example README: it contains `code blocks` that probably need to be updated
+
+Notes:
+
+- We are evaluating options to auto-document as much as possible. Tables on docs/terraformoptions.adoc may be the first point will we address.
+- Any change to README should be reflected on both the markdown and adoc version of the file. We are also evaluating how to reduce friction here.
```

Comment:
Consider: any changes made to `README.md` must be done in `README.adoc` and vice-versa. Again, user engagement could be considered for methods, but I reckon a GitHub Action on push of your canonical source is the simplest.

Reply:
Sentence updated as suggested.
Interested to see how we can leverage GitHub Actions to handle such housekeeping activities, I am not familiar with Actions yet.